src/reportWebVitals.ts

```diff
@@ -1,15 +0,0 @@
-import { ReportHandler } from 'web-vitals';
```

Comment:
is this ok to remove? not sure if this are metrics

Reply:
We're not using it anywhere, so I removed it